homeassistant/components/matter/lock.py

```diff
@@ -168,6 +168,9 @@ def _update_from_device(self) -> None:
             self._attr_is_jammed = (
                 door_state is clusters.DoorLock.Enums.DoorStateEnum.kDoorJammed
             )
+            self._attr_is_open = (
+                door_state is clusters.DoorLock.Enums.DoorStateEnum.kDoorOpen
+            )
```

Comment:
Does this match the meaning of the feature? It should indicate if the door latch is retracted. It is not if the door is physically ajar.

Reply:
Yes this is ok

Reply:
I misunderstood the state. If open state indicates a retracted latch, then I agree, it doesn't match and door open/close state should be exposed with a binary sensor entity.

Reply:
I had to go back to the arch discussion to understand it. The dev docs where unclear :)